koans/ComplexNumberKoans.jl

```diff
@@ -1,3 +1,39 @@
 module ComplexNumberKoans
+#=
+the functions expect complex numbers with the syntax x = 1 + 1im
+=#
 
-end
\ No newline at end of file
+function sum(x,y)
+end
+
+function substract(x,y)
+end
+
+function multiplication(x,y)
+end
+
+function division(x,y)
+end
+
+function power(x, y)
+end
+
+function square_root(x)
+end
+
+function real_part(x)
+end
+
+function imaginary_part(x)
+end
+
+function conjugate(x)
```

Comment:
Although the basic operations are obvious, not everyone solving these exercises may be familiar with complex numbers, and concepts such as the conjugate may be unfamiliar. Could you add a very short explanation of the concept, and perhaps a link to the wikipedia page of complex numbers in the module intro (or something similiar)?